docs/v3/codecs/blosc/v1.0.rst

```diff
@@ -70,33 +70,36 @@ clevel:
     level is 0.
 
 shuffle:
-    An integer value in the set {0, 1, 2, -1} indicating the way
-    bytes or bits are rearranged, which can lead to faster
-    and/or greater compression. A value of 1
-    indicates that byte-wise shuffling is performed prior to
-    compression. A value of 2 indicates the bit-wise shuffling is
-    performed prior to compression. If a value of -1 is given,
-    then default shuffling is used: bit-wise shuffling for buffers
-    with item size of 1 byte, byte-wise shuffling otherwise.
-    Shuffling is turned off completely when the value is 0.
+    Specifies the type of shuffling to perform, if any, prior to compression.
+    Must be one of:
+
+    - ``null``, to indicate no shuffling;
+    - ``"byte"``, to indicate byte-wise shuffling;
+    - ``"bit"``, to indicate bit-wise shuffling.
```

Comment:
I'm in favor of those changes :+1:

Reply:
Looking at the blosc API more now, I think these should probably be changed to be consistent with what the blosc API uses:

"noshuffle", "shuffle", "bitshuffle"